src/commands/synthetics/README.md

```diff
@@ -343,39 +481,380 @@ Your test files must be named with a `.synthetics.json` suffix.
-| `startUrlSubstitutionRegex`        | String           | The regex to modify the starting URL of the test (for browser and HTTP tests only), whether it was given by the original test or the configuration override `startUrl`. <br><br>If the URL contains variables, this regex applies after the interpolation of the variables. <br><br>There are two possible formats: <br><br>- `your_regex\|your_substitution`: The pipe-based syntax, to avoid any conflicts with `/` characters in URLs. For example, `https://example.com(.*)\|http://subdomain.example.com$1` to transform `https://example.com/test` to `http://subdomain.example.com/test`. <br>- `s/your_regex/your_substitution/modifiers`: The slash syntax, which supports modifiers. For example, `s/(https://www.)(.*)/$1extra-$2/` to transform `https://www.example.com` into `https://www.extra-example.com`. |
-| `testTimeout`                      | Number           | The maximum duration of a browser test in seconds.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
-| `variables`                        | Object           | The variables to replace in the test. This object should contain key as the name of the variable to replace and values as the new value of the variable to replace. <br><br> With `--override`, use `variables.NAME=VALUE`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+#### `allowInsecureCertificates` (Boolean)
+
+Disable certificate checks in Synthetic API and Browser tests.
+
+Configuration options:
+
+* Test Config: `"allowInsecureCertificates": true`
```

Comment:
The namings `Global Config` and `Test Config` (with those uppercase letters, and abbreviation) are only internal, it would be better to align and say the same thing everywhere

Reply:
My bad, I see you used it everywhere already. Could we make it explicit somewhere that `Global Config` is short for "global configuration file", and "Test Config" is short for "test (configuration) file"?

It's important to me that we use the same namings everywhere